migrations/versions/0342_org_billing_details.py

```diff
@@ -0,0 +1,73 @@
+"""
+
+Revision ID: 0342_org_billing_details
+Revises: 0341_new_letter_rates
+Create Date: 2021-02-01 14:40:14.809632
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0342_org_billing_details'
+down_revision = '0341_new_letter_rates'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('organisation', sa.Column('billing_contact_email_addresses', sa.Text(), nullable=True))
+    op.add_column('organisation', sa.Column('billing_contact_names', sa.Text(), nullable=True))
+    op.add_column('organisation', sa.Column('billing_reference', sa.String(length=255), nullable=True))
+    op.add_column('organisation', sa.Column('notes', sa.Text(), nullable=True))
+    op.add_column('organisation', sa.Column('purchase_order_number', sa.String(length=255), nullable=True))
+    op.alter_column('rates', 'rate',
```

Comment:
I can see why the organisation columns are being added, but were the changes to the rates and services table intentional?

Reply:
To services yes, to rates no, thanks for spotting! :)

Reply:
It would be great to do something about this ever growing number of things auto-added to migration file. Just I'm not sure what that would entail. Maybe a discussion subject for one of our afternoon cuddles ☺️

Reply:
The commit messages and PR description don't mention `services` so I'm not sure what the changes to the `services` table are for. Could you edit the commit message to explain what's being changed in this table and why?

Reply:
ok, no worries 👍🏼

Reply:
Done now, I also changed migration number to accommodate a migration that just got merged into master.

Reply:
Thanks 🙂